### PR TITLE
Issue #6265: Remove browser-specific CSS prefixes for border-radius, box-sizing, and box-shadow.

### LINKED ISSUE #6265: Remove -moz-border-radius and -webkit-border-radius from core files
## Description of the need

The use of `-moz-border-radius` and `-webkit-border-radius` generate numerous errors visible in developer tools as `Unknown property`. They have been removed from newer versions of Firefox and Chrome browsers respectively. 

Removing these from core files will reduce excessive error messages in developer tools.

Is it time to remove them from core files or is it necessary to keep them to support older browsers?

## Additional information

This relates to 18 files in core:
```
core/misc/dropbutton.css
core/modules/contextual/css/contextual.css
core/modules/node/css/node.preview.css
core/modules/simpletest/files/css_test_files/css_input_without_import.css
core/modules/simpletest/files/css_test_files/css_input_without_import.css.optimized.css
core/modules/simpletest/files/css_test_files/css_input_without_import.css.unoptimized.css
core/modules/system/css/menudropdown.theme.css
core/modules/system/css/menudropdown.theme.breakpoint-queries.css
core/modules/system/css/system.theme.css
core/modules/views_ui/css/views_ui.admin.theme.css
core/themes/bartik/color/preview.css
core/themes/bartik/css/editor.css
core/themes/bartik/css/style.css
core/themes/basis/css/component/admin-tabs.css
core/themes/seven/css/jquery.ui.theme.css
core/themes/seven/css/responsive-tabs.css
core/themes/seven/css/style.css
core/themes/seven/css/views-admin.seven.css
```

The `-moz-border-radius` styling was dropped from the Firefox browser a long time ago, as was `-webkit-border-radius` from the Chrome browser. The latter is present in 13 core files.

These can be replaced with the `border-radius` styling.

Should these be changed or kept to support old browsers?

### PR DESCRIPTION
**Pull request by quicksketch**
Fixes https://github.com/backdrop/backdrop-issues/issues/6265

### CODE REVIEW

**Comment by albanycomputers** [comment]
Are the vendor prefixes still needed for user-select ?

``` css
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
```

Just searching the CSS files for -moz and also found these...

-moz-linear-gradient
-moz-focus-inner
-moz-focusring

-webkit-color-swatch - "This feature is non-standard and should not be used without careful consideration. See full reference on [MDN Web Docs](https://developer.mozilla.org/docs/Web/CSS/::-moz-color-swatch). 


in file "core\modules\simpletest\files\css_test_files\css_input_without_import.css" line 33
```
some :pseudo .thing {
  -moz-border-radius: 3px;
  -webkit-border-radius: 3px;
  border-radius: 3px;
```

in file "\core\themes\seven\css\style.css" line 1099
-moz-transition


From what I can tell, they are not... but I'm no expert :)

Hope this is relevant and helps.

**Comment by quicksketch** [comment]
> Are the vendor prefixes still needed for user-select ?

[According to MDN, Safari still needs the vendor prefix](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select), so maybe we leave it in place until we can remove all them equally?

> Just searching the CSS files for -moz and also found these...

[-moz-focus-inner](https://developer.mozilla.org/en-US/docs/Web/CSS/::-moz-focus-inner) is still required. I couldn't find docs on `-moz-focusring`, but it's part of normalize.css, which I'm hesitant to change.

`-moz-transition` (and other prefixes) _seem_ to be safe to remove but I'm not 100% sure.

File: core/themes/seven/css/style.css [hunk 1]
```diff
@@ -1128,13 +1094,8 @@ textarea.form-textarea {
...
   color: #444b53;
   background: #fff;
-  -webkit-border-radius: 4px;
-  -moz-border-radius: 4px;
   border-radius: 4px;
-  -webkit-transition: border-color 0.25s ease-in-out;
-  -moz-transition: border-color 0.25s ease-in-out;
-  -ms-transition: border-color 0.25s ease-in-out;
-  -o-transition: border-color 0.25s ease-in-out;
+  transition:border-color 0.25s ease-in-out;
```
Comment (indigoxela) [inline core/themes/seven/css/style.css]: Please add one space after the colon. :wink:

Comment (quicksketch) [inline core/themes/seven/css/style.css]:
```suggestion
  transition: border-color 0.25s ease-in-out;
```

**Comment by laryn** [comment]
Thanks @quicksketch et al! Merged into `1.x` for `1.27.0`.

**Pull request merged**